lib/algo/DMD.cpp

```diff
@@ -548,7 +548,7 @@ DMD::projectInitialCondition(const Vector* init, double t_offset)
     *d_phi_imaginary_squared -= *d_phi_imaginary_squared_2;
 
     double* inverse_input = new double[d_phi_real_squared->numRows() *
-                                       d_phi_real_squared->numColumns() * 2];
+                                                                     d_phi_real_squared->numColumns() * 2];
```

Comment:
Is it an effect of the astyle? Perhaps we would like the origin one without too many spaces better.

Reply:
Yeah this is executed by `astyle` version 3.1. I think we can tune some astyle setup to avoid this. I'll make this change in the next PR #226 .

Reply:
This change is due to a difference in astyle 2.05 vs. 3.1/3.2. We do not have an official version of astyle.